semconv/src/main/java/io/opentelemetry/semconv/ResourceAttributes.java

```diff
@@ -1013,242 +1514,649 @@ public final class ResourceAttributes {
    * Deprecated, use the {@code otel.scope.version} attribute.
    *
    * @deprecated Deprecated, use the `otel.scope.version` attribute.
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.OtelIncubatingAttributes#OTEL_LIBRARY_VERSION}
+   *     attribute.
    */
   @Deprecated
   public static final AttributeKey<String> OTEL_LIBRARY_VERSION = stringKey("otel.library.version");
 
-  /** Container labels, {@code <key>} being the label name, the value being the label value. */
+  /**
+   * Container labels, {@code <key>} being the label name, the value being the label value.
+   *
+   * @deprecated deprecated in favor of {@see
+   *     io.opentelemetry.semconv.incubating.ContainerIncubatingAttributes#CONTAINER_LABELS}
+   *     attribute.
+   */
   @Deprecated
   public static final AttributeKeyTemplate<String> CONTAINER_LABELS =
       stringKeyTemplate("container.labels");
 
   // Enum definitions
   @Deprecated
   public static final class CloudPlatformValues {
-    /** Alibaba Cloud Elastic Compute Service. */
+    /**
+     * Alibaba Cloud Elastic Compute Service.
+     *
+     * @deprecated deprecated in favor of {@see
+     *     io.opentelemetry.semconv.incubating.AlibabaIncubatingAttributes.CloudPlatformValues#ALIBABA_CLOUD_ECS}
```

Comment:
sorry, these inner class names have just changed in #83

Reply:
Done. I renamed all occurences to the CloudIncubatingAttributes